target/linux/ramips/base-files/etc/uci-defaults/03_wireless

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (C) 2021 OpenWrt.org
+#
+
+. /lib/functions.sh
+
+board=$(board_name)
+
+case "$board" in
+sony,wg-c10)
+	uci set wireless.radio0.disabled=0
```

Comment:
This will not be accepted. Please remove the file and stick to the common defaults.